pagopa/io-dev-api-server
# chore: migrate `codecov` from `bash` to `github action`

## Short description
This PR migrates `codecov` from `bash` to `github action`.

## How to test
Run static check. See https://github.com/pagopa/io-dev-api-server/actions/runs/8892023803/job/24415706764
